### DISCUSSION TITLE
Ausgeblendete Seitenleiste einblenden

### DISCUSSION THREAD

**Discussion by robotsox** [Zefau/ioBroker.jarvis#2116] [Q&A] [answered]
Hallo,

ich habe über die Webvisu alle Menüleisten ausgeblendet.
Nun komme ich aber auch nicht mehr in die "Einstellungen" rein, da das Burger Menu ja weg ist.

Wie kann ich dieses nun wieder einblenden?

**Comment by mcuiobroker** [comment, marked as answer]
F2
